### ISSUE TITLE
FinalInAbstractClassRule complains not final method in interface

### ISSUE THREAD

**Issue by mimmi20** [localheinz/phpstan-rules#131]
Starting with version 0.12 an check failed on an interface.

#### Steps required to reproduce the problem

```php
<?php
/**
 * This file is part of the ios-build package.
 *
 * Copyright (c) 2019, Thomas Mueller <mimmi20@live.de>
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 */

declare(strict_types = 1);
namespace IosBuild;

interface IosBuildInterface
{
    /**
     * Takes iOS build code and returns corresponding iOS version.
     *
     * @param string $needle the build code
     *
     * @throws BuildException
     * @throws NotFoundException
     *
     * @return string JSON encoded string
     */
    public function getVersion(string $needle): string;
}
```
2. run PHPStan including this library against this file.

#### Expected Result

```shell
 [OK] No errors
```

#### Actual Result

```shell
 ------ -------------------------------------------------------------------------------------------------------------------------
  Line   IosBuildInterface.php
 ------ -------------------------------------------------------------------------------------------------------------------------
  26     Method IosBuild\IosBuildInterface::getVersion() is not final, but since the containing class is abstract, it should be.
 ------ -------------------------------------------------------------------------------------------------------------------------


 [ERROR] Found 1 error

```

**Comment by localheinz** [comment]
@mimmi20 

Thank you for your report, I will fix it immediately!

**Comment by localheinz** [comment]
@mimmi20 

Apologies for the troubles and thanks again for the report - a fix has been released with [`0.12.2`](https://github.com/localheinz/phpstan-rules/releases/tag/0.12.2)!

**Comment by mimmi20** [comment]
Thanks
